gym/vector/sync_vector_env.py

```diff
@@ -123,26 +105,16 @@ def reset_wait(
                 kwargs["seed"] = single_seed
             if options is not None:
                 kwargs["options"] = options
-            if return_info is True:
-                kwargs["return_info"] = return_info
 
-            if not return_info:
-                observation = env.reset(**kwargs)
-                observations.append(observation)
-            else:
-                observation, info = env.reset(**kwargs)
-                observations.append(observation)
-                infos = self._add_info(infos, info, i)
+            observation, info = env.reset(**kwargs)
```

Comment:
Do we need to use kwargs?

Reply:
going to hold off on this unless specifically requested, in accordance with our discussion about kwargs on the other comment.